ferretdb/ferretdb_test.go

```diff
@@ -85,3 +90,52 @@ func Example_unix() {
 
 	// Output: mongodb://%2Ftmp%2Fferretdb-27017.sock
 }
+
+func Example_tls() {
+	certPath := path.Join("..", "build", "certs", "server-cert.pem")
+	keyPath := path.Join("..", "build", "certs", "server-key.pem")
+
+	f, err := New(&Config{
+		Listener: ListenerConfig{
+			TLS:         "127.0.0.1:47017",
+			TLSCertFile: certPath,
+			TLSKeyFile:  keyPath,
+		},
+		Handler:       "pg",
+		PostgreSQLURL: "postgres://postgres@127.0.0.1:5432/ferretdb",
+	})
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	ctx, cancel := context.WithCancel(context.Background())
+
+	done := make(chan struct{})
+
+	go func() {
+		log.Print(f.Run(ctx))
+		close(done)
+	}()
+
+	uri := f.MongoDBURI()
+	fmt.Println(uri)
```

Comment:
Let's use `log.Print(...)`

Reply:
Go testing [examples](https://pkg.go.dev/testing#hdr-Examples) (`// Output: mongodb://127.0.0.1:47017` line below) check stdout, not stderr